src/components/PlaylistForm.vue

```diff
@@ -0,0 +1,220 @@
+              v-model="newPlaylist.name"
+              :rules="[(v) => !!v || $t('errors.playlist.name-blank')]"
+              required
+            />
+            <VTextarea
+              :label="$t('common.description')"
+              rows="3"
+              v-model="newPlaylist.description"
+            />
+            <VAutocomplete
+              :items="playlistTypes"
+              :label="$t('music.playlist.playlist_type')"
+              v-model="newPlaylist.playlist_type"
+              :disabled="hasItems"
+            />
+            <VAutocomplete
+              :items="accessOptions"
+              :label="$t('music.playlist.access')"
+              v-model="newPlaylist.access"
+            />
+            <VBtn
+              :disabled="!isValid"
+              color="primary"
+              class="ma-2"
+              type="submit"
+            >
+              {{
+                this.playlist
+                  ? $t("music.playlist.update")
+                  : $t("music.playlist.create")
+              }}
+            </VBtn>
+          </VCol>
+        </VRow>
+        <VRow no-gutters align="center" justify="center">
+          <VCol md="9" sm="10" cols="12" @change.once="isDirty = true">
+            <h4 class="text-h6 mt-6 ml-4">
+              {{ $tc("music.playlist.items", 2) }}
+            </h4>
+            <VSimpleTable>
+              <thead>
+                <tr>
+                  <th style="width: 1px" class="text-center">Sort</th>
+                  <th>
+                    {{
+                      newPlaylist.playlist_type === "artist"
+                        ? $t("music.name")
+                        : $t("music.title")
+                    }}
+                  </th>
+                  <th></th>
+                </tr>
+              </thead>
+              <Draggable
+                tag="tbody"
+                v-model="newPlaylist.item_ids"
+                handle="[data-draggable=handle]"
+              >
+                <tr
+                  v-for="(item_id, index) of newPlaylist.item_ids"
+                  :key="item_id"
+                >
+                  <td class="text-no-wrap">
+                    <VBtn small icon text class="" data-draggable="handle">
+                      <VIcon>mdi-drag-horizontal-variant</VIcon>
+                    </VBtn>
+                    {{ index }}
+                  </td>
+                  <td class="play-queue__cell">
+                    {{
+                      newPlaylist.playlist_type === "artist"
+                        ? items[item_id].name
+                        : items[item_id].title
+                    }}
+                  </td>
+                  <td class="text-right">
+                    <VBtn
+                      small
+                      icon
+                      text
+                      class="ma-2 red--text"
+                      @click="() => newPlaylist.item_ids.splice(index, 1)"
```

Comment:
This edits the actual item_ids array of the playlist (playlist is probably shallow copied?)